Challenge03/css/flor.css

```diff
@@ -0,0 +1,211 @@
+* {

```

Comment:
Using the * selector with box-sizing: border-box or including a CSS reset library like normalize.css can help achieve consistent and predictable box-sizing behavior across different elements in your web page.